fwprovider/test/resource_metrics_server_test.go

```diff
@@ -0,0 +1,176 @@
+//go:build acceptance || all
+
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package test
```

Comment:
It would be better to move this test file to /fwprovider/cluster/metrics, so it will stay closer to the code.

The other tests in /fwprovider/test are mostly for legacy SDK-based resources/datasources. It is mostly for convenience, as I'd want to keep all acceptance tests under the same root.

Reply:
moved

i see, i think i initially lagged with import cycle but changing package name fix it and it can be with code